site/public/js/course-materials.js

```diff
@@ -0,0 +1,170 @@
+/* global IS_INSTRUCTOR, toggleCMFolders, toggleCMFolder */
+
+// eslint-disable-next-line no-unused-vars
+function setFolderCookie(folderPath, id, open) {
+    const folderData = JSON.parse(Cookies.get('cm_folder_data') || '{}');
+    folderData[folderPath] = { id, open };
+    Cookies.set('cm_folder_data', JSON.stringify(folderData));
+}
+
+// eslint-disable-next-line no-unused-vars
+function toggleFoldersOpen() {
+    Cookies.remove('cm_folder_data');
+    Cookies.set('foldersOpen', toggleCMFolders());
+}
+
+function handleHideMaterialsCheckboxChange(clicked) {
+    const warningIds = {'hide-materials-checkbox': 'upload-form', 'hide-materials-checkbox-edit': 'edit-form', 'hide-folder-materials-checkbox-edit': 'edit-folder-form'};
```

Comment:
This line is pretty long.  It would be better to break it into multiple lines.

Something like this perhaps:
```suggestion
    const warningIds = {
        'hide-materials-checkbox': 'upload-form',
        'hide-materials-checkbox-edit': 'edit-form',
        'hide-folder-materials-checkbox-edit': 'edit-folder-form'
    };
```